xwiki-platform-core/xwiki-platform-mentions/xwiki-platform-mentions-ui/src/main/resources/XWiki/Mentions/MentionsTranslations.xml

```diff
@@ -51,7 +51,8 @@ XWiki.Mentions.MentionsConfigurationClass_mentionsColor=Mentions Color
 XWiki.Mentions.MentionsConfigurationClass_mentionsColor.hint=The color of the mentions to the current user (note: used as the value of a background-color css attribute).
 XWiki.Mentions.MentionsConfigurationClass_selfMentionsColor=Self Mentions Color
 XWiki.Mentions.MentionsConfigurationClass_quoteActivated=Activate the quotes
-XWiki.Mentions.MentionsConfigurationClass_quoteActivated.hint=Defines if a quote of the text surrounding a mention is included in the user mention notifications.</content>
+XWiki.Mentions.MentionsConfigurationClass_quoteActivated.hint=Defines if a quote of the text surrounding a mention is included in the user mention notifications.
+mentions.tip=Mention a user by typing @ then the user name in a Wysiwyg editor.</content>
```

Comment:
`Wysiwyg` why the first letter uppercase? You have example when we use that form in translations? Personally I would write it either all uppercase or all lowercase

Reply:
It should be all uppercase.

Reply:
That's the way it is spelled in the user settings.

Reply:
Then it also needs to be fixed there :)

Reply:
we got https://jira.xwiki.org/browse/XWIKI-11269 about that

Reply:
Oh right :) Fixing.

Reply:
Done